CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.2] - 2018-10-10
+### Added
+- Support `target` parameter for `Scnnr::Client#coordinate`.
```

Comment:
Isn't support for a new parameter `added functionality`? According to Semver this would be a MINOR version change, not a PATCH version: https://semver.org/
-> I think this should become `1.2.0` in that case

Reply:
Thank you for pointing out! I changed the version and push it with force push. Please take a look.